### PR TITLE
created components to show uploaded images and added logic to remove …

### PR DESCRIPTION
**Pull request by 6mahdapihka9**
…old ones

### CODE REVIEW

**Comment by naberalgamma** [comment]
add loader to photos in edit mode

**Review by mazursasha1990 — APPROVED** [review]
I didn`t find any issues, without Yuriy's comment.

File: frontend/src/utils/getFileFromUrl.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    setter((prev: any) => {
+      if (prev.some((f: File) => f.name === name)) {
+        return [...prev];
+      }
+      return [...prev, convertedFile];
+    });
+  });
+}
+
+export default getFileFromUrl;
```
Comment (Yojuks) [inline frontend/src/utils/getFileFromUrl.ts:32]: when you open editing location you see this screen
Screenshot 2022-07-13 at 15 03 05
when you press button upload photo you see this
Screenshot 2022-07-13 at 15 03 14
and when you save you see this
Screenshot 2022-07-13 at 15 03 33
i think that new photo maybe to be after previous not before after submit 
i think that new photo should  be after previous not before  - if you think another - can you explaine?

**Review by Yojuks — CHANGES_REQUESTED** [review]

File: frontend/src/utils/getFileFromUrl.ts (hunk 1 above)
Comment (6mahdapihka9) [reply frontend/src/utils/getFileFromUrl.ts:32]: as it should be

**Review by Yojuks — APPROVED** [review]

**Pull request merged**